From 2ccb6fa0ae52981c78f79afa7009ccab11c6703f Mon Sep 17 00:00:00 2001
From: Ming Liu <liu.ming50@gmail.com>
Date: Sat, 15 Jun 2019 05:18:10 +0800
Subject: [PATCH] CMakeLists.txt: support building static uuu

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Ming Liu <liu.ming50@gmail.com>
---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 302f8cb..f23871e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,6 +4,12 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 option(BUILD_DOC "Build documentation" OFF)
+option(BUILD_STATIC "Build static uuu binary" OFF)
+
+if (BUILD_STATIC)
+	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
+	set(CMAKE_EXE_LINKER_FLAGS "-static -static-libgcc -static-libstdc++")
+endif (BUILD_STATIC)
 
 add_subdirectory(libuuu)
 add_subdirectory(uuu)
-- 
2.7.4

