From 9e49ff1c3b9e310ab69b3c487c09aebb8943bcac Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Tue, 16 Dec 2014 11:12:44 +0100
Subject: [PATCH] systemd_machine_id_commit: allow standalone build

---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index ac7924e..514345d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2247,6 +2247,8 @@ systemd_fsck_LDADD = \
 	libsystemd-shared.la
 
 # ------------------------------------------------------------------------------
+generate_sources: $(BUILT_SOURCES)
+.PHONY : generate_sources
 systemd_machine_id_commit_SOURCES = \
 	src/machine-id-commit/machine-id-commit.c \
 	src/core/machine-id-setup.c \
-- 
1.9.3

