From dbafa734d8d6ba4cb5efcbb2260105babcdcde1e Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Sun, 26 Oct 2014 14:01:27 +0100
Subject: [PATCH] networkd: don't stop the dhcp server when link goes down

---
 src/network/networkd-link.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/network/networkd-link.c b/src/network/networkd-link.c
index c6e173f..26ef0fe 100644
--- a/src/network/networkd-link.c
+++ b/src/network/networkd-link.c
@@ -344,15 +344,6 @@ static int link_stop_clients(Link *link) {
                 }
         }
 
-        if (link->dhcp_server) {
-                k = sd_dhcp_server_stop(link->dhcp_server);
-                if (k < 0) {
-                        log_warning_link(link, "Could not stop DHCPv4 server: %s",
-                                         strerror(-r));
-                        r = k;
-                }
-        }
-
         if(link->icmp6_router_discovery) {
 
                 if (link->dhcp6_client) {
