From bea9efea4dae46f66b38b630e461883ac311a9e6 Mon Sep 17 00:00:00 2001
From: Ming Liu <ming.liu@toradex.com>
Date: Wed, 27 Nov 2019 17:02:47 +0100
Subject: [PATCH] rdp: add libwinpr2 dependency

Upstream-Status: Pending

Signed-off-by: Ming Liu <ming.liu@toradex.com>
---
 libweston/backend-rdp/meson.build | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/libweston/backend-rdp/meson.build b/libweston/backend-rdp/meson.build
index 237cd0a..14ad030 100644
--- a/libweston/backend-rdp/meson.build
+++ b/libweston/backend-rdp/meson.build
@@ -9,6 +9,11 @@ if not dep_frdp.found()
 	error('RDP-backend requires freerdp2 which was not found. Or, you can use \'-Dbackend-rdp=false\'.')
 endif
 
+dep_winpr = dependency('winpr2', version: '>= 2.0.0', required: false)
+if not dep_frdp.found()
+        error('RDP-backend requires winpr2 which was not found. Or, you can use \'-Dbackend-rdp=false\'.')
+endif
+
 if cc.has_header('freerdp/version.h', dependencies: dep_frdp)
 	config_h.set('HAVE_FREERDP_VERSION_H', '1')
 endif
@@ -24,6 +29,7 @@ endif
 deps_rdp = [
 	dep_libweston_private,
 	dep_frdp,
+	dep_winpr,
 ]
 plugin_rdp = shared_library(
 	'rdp-backend',
-- 
2.7.4

