diff --git a/compositor/screen-share.c b/compositor/screen-share.c
index 33de2b1f..2e3529d5 100644
--- a/compositor/screen-share.c
+++ b/compositor/screen-share.c
@@ -1122,6 +1122,7 @@ WL_EXPORT int
 wet_module_init(struct weston_compositor *compositor,
 		int *argc, char *argv[])
 {
+	struct weston_output *output;
 	struct screen_share *ss;
 	struct weston_config *config = wet_get_config(compositor);
 	struct weston_config_section *section;
@@ -1138,5 +1139,15 @@ wet_module_init(struct weston_compositor *compositor,
 	weston_compositor_add_key_binding(compositor, KEY_S,
 				          MODIFIER_CTRL | MODIFIER_ALT,
 					  share_output_binding, ss);
+
+	output = weston_output_find(compositor,
+				    wl_fixed_to_int(0),
+				    wl_fixed_to_int(0));
+	if (!output) {
+		weston_log("Cannot pick output: Pointer not on any output\n");
+		return;
+	}
+
+	weston_output_share(output, ss->command);
 	return 0;
 }
