diff --git a/src/client/qwaylandwindow.cpp b/src/client/qwaylandwindow.cpp
index ac84e12b..a9ae2ea9 100644
--- a/src/client/qwaylandwindow.cpp
+++ b/src/client/qwaylandwindow.cpp
@@ -707,9 +707,7 @@ bool QWaylandWindow::createDecoration()
             }
         }
     }
 
-    if (!mDisplay->supportsWindowDecoration())
-        return false;
 
     static bool decorationPluginFailed = false;
     bool decoration = false;
