diff --git a/src/client/qwaylandwindow.cpp b/src/client/qwaylandwindow.cpp
index c46c498..4e35a47 100644
--- a/src/client/qwaylandwindow.cpp
+++ b/src/client/qwaylandwindow.cpp
@@ -708,9 +708,6 @@ void QWaylandWindow::setWindowFlags(Qt::WindowFlags flags)
 
 bool QWaylandWindow::createDecoration()
 {
-    if (!mDisplay->supportsWindowDecoration())
-        return false;
-
     static bool decorationPluginFailed = false;
     bool decoration = false;
     switch (window()->type()) {
