diff --git a/src/src.pro b/src/src.pro
index 4ecbc71..2897b8f 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -13,6 +13,21 @@ qtConfig(wayland-client) {
     sub_client.target = sub-client
     SUBDIRS += sub_client
 
+    sub_platformplugins.subdir = plugins/platforms
+    sub_platformplugins.depends = sub-client
+    sub_platformplugins.target = sub-platformplugins
+    SUBDIRS += sub_platformplugins
+
+    sub_decorationplugins.subdir = plugins/decorations
+    sub_decorationplugins.depends = sub-client
+    sub_decorationplugins.target = sub-decorationplugins
+    SUBDIRS += sub_decorationplugins
+
+    sub_shellintegrationplugins.subdir = plugins/shellintegration
+    sub_shellintegrationplugins.depends = sub-client
+    sub_shellintegrationplugins.target = sub-shellintegrationplugins
+    SUBDIRS += sub_shellintegrationplugins
+
     qtConfig(wayland-server) {
         sub_compositor.subdir = compositor
         sub_compositor.depends = sub-qtwaylandscanner
@@ -24,7 +39,7 @@ qtConfig(wayland-client) {
         sub_imports.target = sub-imports
         SUBDIRS += sub_imports
 
-        sub_plugins.subdir = plugins
+        sub_plugins.subdir = plugins/hardwareintegration
         sub_plugins.depends = sub-qtwaylandscanner sub-client sub-compositor
         sub_plugins.target = sub-plugins
         SUBDIRS += sub_plugins
