diff --git a/tests/auto/auto.pro b/tests/auto/auto.pro
index 79ad29bd..ba02a52b 100644
--- a/tests/auto/auto.pro
+++ b/tests/auto/auto.pro
@@ -2,6 +2,6 @@ TEMPLATE=subdirs
 QT_FOR_CONFIG += waylandclient-private
 
 qtConfig(wayland-client): \
-    SUBDIRS += client cmake
+    SUBDIRS += cmake
 qtHaveModule(waylandcompositor): \
     SUBDIRS += compositor
diff --git a/tests/auto/client/client.pro b/tests/auto/client/client.pro
index 916b3abe..e69de29b 100644
--- a/tests/auto/client/client.pro
+++ b/tests/auto/client/client.pro
@@ -1,3 +0,0 @@
-TEMPLATE=subdirs
-
-SUBDIRS += client
