diff --git a/tests/auto/auto.pro b/tests/auto/auto.pro
index 79ad29b..ba02a52 100644
--- a/tests/auto/auto.pro
+++ b/tests/auto/auto.pro
@@ -2,6 +2,6 @@ TEMPLATE=subdirs
 QT_FOR_CONFIG += waylandclient-private
 
 qtConfig(wayland-client): \
-    SUBDIRS += client cmake
+    SUBDIRS += cmake
 qtHaveModule(waylandcompositor): \
     SUBDIRS += compositor
diff --git a/tests/auto/client/client.pro b/tests/auto/client/client.pro
deleted file mode 100644
index 9fd8fc3..0000000
--- a/tests/auto/client/client.pro
+++ /dev/null
@@ -1,7 +0,0 @@
-TEMPLATE=subdirs
-
-SUBDIRS += \
-    client \
-    iviapplication \
-    xdgshellv6 \
-    wl_connect
